sdk/pubkey/src/lib.rs

```diff
@@ -161,13 +266,18 @@ impl TryFrom<Vec<u8>> for Pubkey {
     }
 }
 
+#[cfg(feature = "std")]
```

Comment:
Is this needed? It should be ok in no-std contexts, just like the `TryFrom<&[u8]>` impl